WooCommerce/Classes/ViewRelated/Products/Edit Product/Inventory Settings/ProductInventorySettingsViewController.swift

```diff
@@ -201,6 +198,36 @@ extension ProductInventorySettingsViewController {
 private extension ProductInventorySettingsViewController {
     func handleSKUChange(_ sku: String?) {
         self.sku = sku
+
+        // If the sku is identical to the old one, is always valid.
+        guard sku != product.sku else {
+            skuIsValid = true
+            hideError()
```

Comment:
do you know if there is an outstanding validation job in the throttler, it needs to be canceled?